Add page for HTTP Refresh header (#35709)
mdn/content: files/en-us/web/html/element/meta/index.md

        Also see {{HTTPHeader("Content-Security-Policy")}}.

    - `content-type`

      - : Declares the [MIME type](/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) and the document's character encoding. The `content` attribute must have the value `"text/html; charset=utf-8"` if specified. This is equivalent to a `<meta>` element with the [`charset`](#charset) attribute specified and carries the same restriction on placement within the document. **Note:** Can only be used in documents served with a `text/html` — not in documents served with an XML MIME type.

        Also see {{HTTPHeader("Content-Type")}}.

    - `default-style`

      - : Sets the name of the default [CSS style sheet](/en-US/docs/Web/CSS) set.

    - `x-ua-compatible`

      - : If specified, the `content` attribute must have the value `"IE=edge"`. User agents are required to ignore this pragma.

    - `refresh`

      - : This instruction specifies:

>         - The number of seconds until the page should be reloaded - only if the [`content`](#content) attribute contains a non-negative integer.

Reviewer: Mabye
```suggestion
        - The number of seconds until the page should be reloaded - only if the [`content`](#content) attribute starts with a non-negative integer.
```

Author: What I meant is "`content` _is_ a non-negative integer", i.e. you shouldn't have anything else.

Reviewer: What about when it has a value like your example. "3;url=https://www.mozilla.org""

Author: That would be the next bullet, since that would be a redirect, not a reload.

>         - The number of seconds until the page should be reloaded - only if the [`content`](#content) attribute contains a non-negative integer.

Author: ```suggestion
        - The number of seconds until the page should be reloaded - if the [`content`](#content) attribute's value is a non-negative integer.
```

>         - The number of seconds until the page should redirect to another - only if the [`content`](#content) attribute contains a non-negative integer followed by the string '`;url=`', and a valid URL.

Author: ```suggestion
        - The number of seconds until the page should redirect to another - if the [`content`](#content) attribute's value is a non-negative integer followed by  `;url=` and a valid URL.
```
